examples/wrapper-types/src/lib.rs

```diff
@@ -1,8 +1,9 @@
-// We use `JsonObject` in our UDL. It moves to and from Uniffi bindings via a string.
-pub type JsonObject = serde_json::Value;
+// Trivial "Handle" type which wraps an i64.
+pub struct Handle(pub i64);
 
-// We must implement the UniffiCustomTypeWrapper trait.
-impl UniffiCustomTypeWrapper for JsonObject {
+// We must implement the UniffiCustomTypeWrapper trait for each type that we want to wrap on the
+// rust side
+impl UniffiCustomTypeWrapper for serde_json::Value {
```

Comment:
I must be missing something here - how does `JsonObject` in the UDL work now that you've removed the type alias?